### PR TITLE
church model

### PR DESCRIPTION
**Pull request by LeeMoTakito**
Please check for me thank you ;)

### CODE REVIEW

File: src/module/church.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+export class Church extends Model<InferAttributes<Church>, InferCreationAttributes<Church>>{
+    declare churchId: number;
+    declare churchName: string;
+    declare password: string;
+    declare email: string;
+    declare address: string;
+    declare phoneNumber: number;
+    declare statementOfFaith: string;
+    declare welcomeMessage: string;
+    declare serviceTime: number;
```
Comment (RylanTech) [inline src/module/church.ts]: We might want to change serviceTime to a string

**Comment by LeeMoTakito** [comment]
I changed

**Review by RylanTech — CHANGES_REQUESTED** [review]
You probably want to change serviceTime to string instead of number

**Review by akinsman22 — APPROVED** [review]
I see the change. It looks good as string

**Pull request merged**